added possibility to trigger tab reload from outside

akosbordas opened the pull request:
#162 I created a modification that I mentioned in the issue

Commit:
added possibility to trigger tab reload from outside

webbiesdk commented:
The thing is, with this suggestion, just executing `reload = true` will only reload it the first time. 

How about the directive resets the value back to `false`, after it has reloaded the tabs?

Commit:
reload flag is reset after it was raloaded

Commit:
check true in reload watch instead of comparing with the old value

akosbordas commented:
I was thinking about it too. But it seemed more explicit to manage this flag from outside. The caller have to keep a boolean property on the scope anyway. I thought it can be more transparent if this is switched only one side. But I'm ok with your suggestion, so I made it work. I also fixed a bug because I replaced the old and the new value of the watch in my first commit.

webbiesdk commented:
Finally had time to look at it, looked good.
